src/v2/Apps/Partner/Components/PartnerHeader/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react"
+import styled from "styled-components"
+import {
+  Box,
+  Text,
+  Image,
+  color,
+  GridColumns,
+  Column,
+  Flex,
+} from "@artsy/palette"
+import { PartnerHeaderAddress } from "./PartnerHeaderAddress"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FollowProfileButtonFragmentContainer as FollowProfileButton } from "v2/Components/FollowButton/FollowProfileButton"
+import { useSystemContext } from "v2/Artsy"
+import { ContextModule } from "@artsy/cohesion"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { PartnerHeader_partner } from "v2/__generated__/PartnerHeader_partner.graphql"
+
+export interface PartnerHeaderProps {
+  partner: PartnerHeader_partner
+}
+
+export const HeaderImage = styled(Image)`
+  border: 1px solid ${color("black10")};
+`
+
+export const PartnerHeader: React.FC<PartnerHeaderProps> = ({ partner }) => {
+  const { user } = useSystemContext()
+  const partnerIconUrl = partner.profile?.icon?.url
+  const hasLocations = partner.locations?.totalCount > 0
+  const canFollow =
+    partner && partner.type !== "Auction House" && !!partner.profile
+
+  return (
+    <GridColumns gridRowGap={2} my={[2, 4]}>
+      <Column span={[12, 10]}>
+        <Flex>
+          {partnerIconUrl && (
+            <Flex mr={2}>
+              <RouterLink
+                to={partner.href}
+                style={{ display: "flex", textDecoration: "none" }}
+              >
+                <HeaderImage
+                  src={partnerIconUrl}
+                  alt={partner.name}
+                  width={[60, 80]}
+                  height={[60, 80]}
+                />
+              </RouterLink>
+            </Flex>
+          )}
+          <Box>
+            <Text variant="largeTitle">
+              <RouterLink
+                style={{
+                  textDecoration: "none",
+                }}
+                to={partner.href}
+              >
+                {partner.name}
+              </RouterLink>
+            </Text>
+            {hasLocations && (
+              <Text color={color("black60")} variant="text">
+                <PartnerHeaderAddress {...partner.locations} />
+              </Text>
+            )}
+          </Box>
+        </Flex>
+      </Column>
+      <Column span={[12, 2]}>
+        {canFollow && (
+          <FollowProfileButton
```

Comment:
Could you please help me with `FollowProfileButton`? The sign-up modal is not opening when an unauthorized user clicks on the button. As far as I see the problem is here: https://github.com/artsy/force/blob/725fe09a32c9b3e4c634c44014d9a272fa0743e6/src/novo/src/client.tsx#L26-L31
Analytics attaching stops with error and the function can't call `initModalManager`. How can I solve this problem?

Reply:
I'm not familiar with this problem; is this an issue on other pages? Sounds possibly novo related cc @damassi @icirellik ?

Reply:
What's ther error @kiryl-zubarau?

Reply:
Here is an screenshot @damassi:
<img width="966" alt="image" src="https://user-images.githubusercontent.com/79979820/112025707-7f05f300-8b46-11eb-9272-40c76a44bad0.png">

Reply:
Interesting -- yeah that `window.analytics` should be there, and its concerning if it's not. Likely related to novo. 

I'll pull down this branch and take a quick look 👍

Reply:
Thanks

Reply:
Hm! I wonder if there might be a missing ENV var, or a race condition of some kind (cc @sweir27). Everything looks good over here: 

![partner](https://user-images.githubusercontent.com/236943/112034513-cca93e00-8afb-11eb-853c-04dae046b1c8.gif)

Reply:
It's strange because login button doesn't work on staging for the page. https://staging.artsy.net/partner2/athr

Reply:
Oh I know what it is. We're migrating our `novo` pages over and as an interim step need to add the `partner2` path to this array: https://github.com/artsy/force/blob/7fe75bbd1151d99c3c5e087daf9d31d6095a8897/src/desktop/lib/webpackPublicPath.ts#L44

Reply:
@kiryl-zubarau it may be that our segment lib can't initialize properly-- do you have `SEGMENT_WRITE_KEY` set? If not, you can pull the value from staging (`sd.SEGMENT_WRITE_KEY` in the browser console) and try it with that present.

Reply:
Added the path to the array, hope this helps for `staging`. Thanks @damassi !

Reply:
Worked! Thanks @sweir27 !